### PR TITLE
Updated helm dependencies and added metrics-server chart

### PR DESCRIPTION
**Pull request by tartale**
https://aspenmesh.atlassian.net/browse/AM-4677

### CODE REVIEW

File: charts/aspenmesh-collector/Chart.yaml [hunk 1]
```diff
@@ -16,3 +16,6 @@ dependencies:
   - name: custom-pod-autoscaler-operator
     version: 1.3.0
     repository: ""
+  - name: metrics-server
+    version: 3.8.3
+    repository: https://kubernetes-sigs.github.io/metrics-server
```
Comment (tartale) [inline charts/aspenmesh-collector/Chart.yaml:21]: I'm not sure whether I did this right; I needed to run `helm dependency build` and it added three tar files below

Here's what happens if they're not present:

Screen Shot 2023-01-27 at 1 11 10 PM

Comment (jizh6311) [reply charts/aspenmesh-collector/Chart.yaml:21]: If we are not sure, another way is to run `kubectl apply -f ...` in the one-line script: https://github.com/kubernetes-sigs/metrics-server#Installation?

**Review by jizh6311 — APPROVED** [review]
If this works on your cluster, this approach should be OK

File: charts/aspenmesh-collector/Chart.yaml (hunk 1 above)
Comment (tartale) [reply charts/aspenmesh-collector/Chart.yaml:21]: I wondered about that, but I thought it'd be simpler to do as a dependency; one less thing in the script, one less error to check for...  not sure

**Comment by tartale** [comment]
> If this works on your cluster, this approach should be OK

Yeah - this worked for me; if we need to we can change it later, once we have Rick's input

**Pull request merged**